stock-managment/src/main/java/org/dhis2/android/rtsm/ui/home/HomeViewModel.kt

```diff
@@ -74,10 +74,10 @@ class HomeViewModel @Inject constructor(
     private val _toolbarTitle = MutableStateFlow(TransactionType.DISTRIBUTION)
     val toolbarTitle: StateFlow<TransactionType> get() = _toolbarTitle
 
-    private val _fromFacility = MutableStateFlow("From...")
+    private val _fromFacility = MutableStateFlow("From facility")
     val fromFacility: StateFlow<String> get() = _fromFacility
 
-    private val _deliveryTo = MutableStateFlow<String?>(null)
+    private val _deliveryTo = MutableStateFlow<String?>("To facility")
```

Comment:
Could you extract the hard coded string to string resources?

Reply:
Solved!